lib/tty/box.rb

```diff
@@ -341,7 +341,9 @@ def top_right_corner(border)
     #
     # @api private
     def top_titles_size(title)
-      title[:top_left].to_s.size + title[:top_center].to_s.size + title[:top_right].to_s.size
+      color.strip(title[:top_left].to_s).size   +
+      color.strip(title[:top_center].to_s).size +
+      color.strip(title[:top_right].to_s).size
```

Comment:
`rubocop` will issue a warning if these lines are not indented by two spaces. Would you mind indenting?